config/locales/en.yml

```
@@ -35,3 +35,4 @@ en:
     user_updated: "User updated with the next attributes:\n First name - %{first_name},\n Last name - %{last_name},\n Email - %{email}\n"
     user_logged_in: "User logged in with the next attributes:\n First name - %{first_name},\n Last name - %{last_name}\n"
     user_registered: "New user registered with the next attributes:\n First name - %{first_name},\n Last name - %{last_name}\n"
+    reset_password_request: "User send reset password request"
```

```suggestion
    reset_password_requested: "User requested reset password instructions"
```

Would it be better to name like action?